### PR TITLE
Updates Onboarding page

### LINKED ISSUE #238: Update Onboarding section to include data access
We should add data access to the [Onboarding](https://gstt-csc.github.io/handbook.html#Onboarding/) guide to e..g, enable easier and more robust collaboration on data curation, data checking, etc. vs. exporting CSVs and sharing across email/the Shared drive.

### PR DESCRIPTION
**Pull request by heyhaleema**
This PR:

- Adds MICAD guidance to the Room Booking guide
- Reformats and updates the Onboarding page e.g., to add more details on Citrix setup and access to data platforms

### CODE REVIEW

**Review by robd78 — APPROVED** [review]

File: _handbook/onboarding.md [hunk 1]
```diff
@@ -6,96 +6,106 @@ shorthand: Onboarding
...
-In order to check payslips, p60, pension contributions etc. you need to get access to your Electronic Staff Record.
-   - Browse https://my.esr.nhs.uk/dashboard/web/esrweb
-   - Go to login to ESR
-   - Click Unlock Account if is your first time
-   - Fill in details and wait for an email with instructions. It will usually be your username and a link to set up your password
+New starters should ensure they register and have access to:
 
-**Citrix Access**
+- [MICAD room booking system](https://gstt.nhs.micadipr.net/pages/roomMonitoringClients/rmSignIN.asp?return=%2Fpages%2FroomMonitoringClients%2FrmRoomSearchForClients%2Easp%3F)* i.e., to book meeting rooms across GSTT sites.
+- [Electronic Staff Record (EST)](https://my.esr.nhs.uk/dashboard/web/esrweb) i.e., to access payslips, P60 forms, pension contributions, etc.
```
Comment (sophie22) [inline _handbook/onboarding.md:48]: abbreviation should be ESR, not EST

File: _handbook/onboarding.md [hunk 2]
```diff
@@ -6,96 +6,106 @@ shorthand: Onboarding
...
+  - Click on "Unlock Account" to register for the first time.
+- [HR Portal](https://fids.gstt.nhs.uk/adfs/ls/)** i.e., to raise and manage HR requests/transactions, access HR policies, etc.
+- [IT Service Centre](https://gstt.service-now.com/ess/?id=my_requests)** i.e., to raise and manage IT requests/incidents, access the IT system catalogue, etc.
```
Comment (sophie22) [inline _handbook/onboarding.md:51]: I believe this is linked to the GSTT NHS email account which should be requested by the line manager of the new starter, the same GSTT account is used for the IT Service Centre

File: _handbook/onboarding.md [hunk 3]
```diff
@@ -6,96 +6,106 @@ shorthand: Onboarding
...
 
-Citrix allows you to access GSTT network when you are at home. In order to get access you have to:  
-   - Call St Thomas' at 020 7188 7188 and ask to be redirected to IT
-   - Once connected, ask about the steps to get access to Citrix
-   - You will receive an email with detailed instructions
+\* _Accessible only via the Trust VPN i.e., either on-site or whilst logged into Citrix (see below for how to set up this remote desktop and application access)_.
```
Comment (sophie22) [inline _handbook/onboarding.md:53]: IT Service portal is available without Citrix too (once account is set up)

File: _handbook/onboarding.md [hunk 4]
```diff
@@ -6,96 +6,106 @@ shorthand: Onboarding
...
 
-**Get Access to GitHub**
+\** _The usernames used to logged into these websites are either your GSTT email address or GSTT username, and the password used will be the password you use to login to your email and desktop on-site_.
```
Comment (sophie22) [inline _handbook/onboarding.md:55]: typo: to *log* into

File: _handbook/onboarding.md [hunk 5]
```diff
@@ -6,96 +6,106 @@ shorthand: Onboarding
...
+   
+After completing the above steps, you should be able to access your GSTT applications remotely by logging onto Citrix with your GSTT username and password, and OTO generated on the SafeNet Mobile Pass application. If you require access to your GSTT desktop i.e., a single remote desktop window with multiple applications contained therein, and this has not been automatically enabled in your Citrix, please contact IT. 
 
-New starters will be expected to arrange 30 min meetings with each staff member of the CSC, on Teams or in-person, for 
-introductions as early familiarity with the team encourages a supportive working culture. The meetings should discuss:
+**Access to GSTT data platforms**
 
-- Career to date ( new starter and current staff member)
-- Length of new-starters time with the CSC
+Depending on the projects you will be involved in, you may also need to request access for and familiarise yourselves with the following data platforms to collaborate with the rest of the project team.
```
Comment (sophie22) [inline _handbook/onboarding.md:73]: typo?: familiarise *yourself*, beginning of sentence is not plural

File: _handbook/onboarding.md [hunk 6]
```diff
@@ -6,96 +6,106 @@ shorthand: Onboarding
...
   - [Bioinformatics Trainee](blogs/articles/2021/08/25/STP_blog_Igor.html)
   - [Clinical Engineering Trainee](/blogs/articles/2022/07/01/STP_blog_Isabella.html)
-- **Communicate *wanted* experience** - as opposed to *needed* experience, separating out the trainees interests from tasks required by the competencies/CSC workload.
+- **Communicate *wanted* experience** (as opposed to *needed* experience) to separate out the trainees' interests from tasks required by the competencies and/or CSC workload.
 - **Get assigned 1st and 2nd Supervisor**
 - **Organise weekly check-in sessions with supervisors**
-- **Confirm a schedule for completing projects, competencies, DOPS, ICE's**
-
-[**Get Involved!**](/getinvolved.html) - How to join the CSC Team at GSTT
+- **Confirm a schedule for completing projects, competencies, DOPS, ICE's, etc.**
```
Comment (sophie22) [inline _handbook/onboarding.md:111]: typo: it's called *OCE* for Observed Clinical Event, not ICE

File: _handbook/onboarding.md (hunk 1 above)
Comment (heyhaleema) [reply _handbook/onboarding.md:48]:
```suggestion
- [Electronic Staff Record (ESR)](https://my.esr.nhs.uk/dashboard/web/esrweb) i.e., to access payslips, P60 forms, pension contributions, etc.
```

File: _handbook/onboarding.md (hunk 6 above)
Comment (sophie22) [reply _handbook/onboarding.md:111]: https://nshcs.hee.nhs.uk/programmes/stp/trainees/workbased-assessment/

**Pull request merged**